.deny.toml

```diff
@@ -40,9 +40,6 @@ private = { ignore = true }
 
 [sources]
 allow-git = [
-    # Waiting on releases; used in examples only
```

Comment:
I think we should either add this line back or completely remove `allow-git`. And if we completely remove `allow-git`, I think we should update the title of this PR.

I do think it could be more handy to keep `allow-git`, with this comment, in case it may potentially be needed again in the future.